common/views/components/ContentPage/ContentPage.tsx

```diff
@@ -1,45 +1,41 @@
-// @flow
-import { Children, Fragment, createContext } from 'react';
+import {
+  Children,
+  Fragment,
+  createContext,
+  ReactNode,
+  ReactElement,
+} from 'react';
 import Contributors from '../Contributors/Contributors';
-// $FlowFixMe (tsx)
 import Layout8 from '../Layout8/Layout8';
-// $FlowFixMe (tsx)
 import Layout12 from '../Layout12/Layout12';
 import SeriesNavigation from '../SeriesNavigation/SeriesNavigation';
 import PageHeader from '../PageHeader/PageHeader';
 import Outro from '../Outro/Outro';
 import { classNames } from '../../../utils/classnames';
-import type { Node, Element, ElementProps } from 'react';
-// $FlowFixMe (tsx)
-import Body from '../Body/Body';
 import SpacingSection from '../SpacingSection/SpacingSection';
 import SpacingComponent from '../SpacingComponent/SpacingComponent';
-// $FlowFixMe(tsx)
 import CompactCard from '../CompactCard/CompactCard';
 import Image from '../Image/Image';
-// $FlowFixMe (tsx)
 import Space from '../styled/Space';
-// $FlowFixMe (tsx)
 import { WeAreGoodToGo } from '@weco/common/views/components/CovidIcons/CovidIcons';
 import { prismicPageIds } from '@weco/common/services/prismic/hardcoded-id';
-// $FlowFixMe (tsx)
 import BannerCard from '../BannerCard/BannerCard';
 /*eslint-disable */
 export const PageBackgroundContext = createContext<'cream' | 'white'>('white');
 
 // TODO: use Element<typeof Component>
-type Props = {|
-  id: string,
-  isCreamy?: boolean,
-  Header: Element<typeof PageHeader>,
-  Body: Element<typeof Body>,
+type Props = {
+  id: string;
+  isCreamy?: boolean;
+  Header: ReactElement<typeof PageHeader>;
+  Body: any;
   // This is used for content type specific components e.g. InfoBox
-  children?: ?Node,
-  contributorProps?: ElementProps<typeof Contributors>,
-  Siblings?: Element<typeof SeriesNavigation>[],
-  outroProps?: ?ElementProps<typeof Outro>,
-  seasons?: any, // TODO
-|};
+  children?: ReactNode | null | undefined;
```

Comment:
The type of `ReactNode` is `ReactNode = ReactChild | ReactFragment | ReactPortal | boolean | null | undefined`.

So you shouldn't have to have all these type. You should actually be able to have `children: ReactNode` as far as I can tell.